recipes/libsbml-plus-packages/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "libsbml-plus-packages" %}
```

Comment:
Why this name?

Reply:
There is a (non-maintained) libsbml conda package, in the bioconda channel. My issue is that it was not compiled with the sbml packages (think extra functionalities) I needed. So I did this new conda package, with the core libsbml + all the packages. 
In retrospect, I completely agree that the work package is confusing (even more so since it means different things in the conda and libsbml context). 

I guess this is a very good opportunity to change it to what is should be : libsbml.